Src/BootCamp.Chapter/Address.cs

```diff
@@ -8,17 +8,44 @@ public class Address
     {
         public string Recipient { get; }
         public string Street { get; }
+        public string PostalCode { get; }
 
         public Address(string recipient, string street, string town, string county, string postalCode, string country, string houseNumber = "")
         {
             Recipient = recipient;
             Street = street;
+            PostalCode = postalCode; 
         }
 
         public static bool TryParse(string addressString, out Address address)
         {
             address = default;
-            return false;
+            
+            if (String.IsNullOrEmpty(addressString))
+            {
+                return false; 
+            }
+
+            var splittedAdress = addressString.Split(Environment.NewLine);
+
+            if (splittedAdress.Length != 7)
```

Comment:
7 is a magic number.

Reply:
oke, so I have to make a constant named totalMandotoryProperties which holds 7 ?